content/en/platform/corda/4.12/enterprise/node/operating/tvu/tvu-cli-examples.md

```diff
@@ -0,0 +1,53 @@
+---
+date: '2023-12-20'
+section_menu: corda-enterprise-4-12
+menu:
+  corda-enterprise-4-12:
+    identifier: corda-enterprise-4-12-tvu-cli-examples
+    parent: corda-enterprise-4-12-tvu-cli
+tags:
+- tvu cli
+- tvu
+- transaction validator utility
+title: TVU CLI command examples
+weight: 100
+---
+
+# TVU CLI command examples
+
+The TVU tool streams and validates all the transactions from a provided database. This database can be provided directly by the user using `-d` or `--datasource` CLI option or can be picked up from a `node.conf` file when the node’s base directory is given using the `-b` or `--base-directory` CLI option.
+
```

Comment:
The following CLI command examples assume you have a node installation available with transactions in the database and the associated CorDapps in the node cordapps directory.

Reply:
Added a note